bcda/client/testdata/Patient.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Are these new test files redundant/duplicates of the test files located here: https://github.com/CMSgov/bcda-app/tree/master/shared_files/synthetic_beneficiary_data

Reply:
Yup they are, thinking more about it. Doesn't really buy us anything to keep the `testdata` directory. I'll go ahead and remove it.